swap_meet/vendor.py

```diff
@@ -1,2 +1,96 @@
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return None
+
+    def get_by_id(self, id):
+        for item in self.inventory:
+            if item.id == id:
+                return item
+        return None
+
+    def swap_items(self, other_vendor, my_item, their_item):
+        if (my_item not in self.inventory or their_item not in other_vendor.inventory):
+            return False
+        else:
```

Comment:
small point:

starting a function with `if`... `return` can be considered a "guard".  There's no need for the `else` and indentation, as the only way the code executes is if it didn't go down the `return` path.  This can be cleaner, and minimize extra indentation :)